### PR TITLE
Feature/resources page

### PR DESCRIPTION
**Pull request by greeshmareddyg**
Resource page:

Story: https://github.ibm.com/brand/design-language/issues/29

content changes: https://ibm.invisionapp.com/share/8BNZI1CCHR9#/screens/319338332

### CODE REVIEW

File: src/pages/resources.mdx [hunk 1]
```diff
@@ -0,0 +1,463 @@
...
+
+![](/images//google_fonts.png)
+
+  </ResourceCard>
+</Column>
+</Row>
+
+## Motion
+
+<Row>
```
Comment (alisonjoseph) [inline src/pages/resources.mdx]:
```suggestion
<Row className="resource-card-group">
```

File: src/pages/resources.mdx [hunk 2]
```diff
@@ -0,0 +1,463 @@
...
+
+![](/images/GitHubIcon.svg)
+
+  </ResourceCard>
+</Column>
+</Row>
+
+## Other IBM resources
+
+<Row>
```
Comment (alisonjoseph) [inline src/pages/resources.mdx]:
```suggestion
<Row className="resource-card-group">
```

File: src/pages/resources.mdx [hunk 3]
```diff
@@ -0,0 +1,463 @@
...
+
+![](/images/bee.png)
+
+  </ResourceCard>
+</Column>
+</Row>
+
+## External resources
+
+<Row>
```
Comment (alisonjoseph) [inline src/pages/resources.mdx]:
```suggestion
<Row className="resource-card-group">
```

File: src/pages/resources.mdx [hunk 4]
```diff
@@ -0,0 +1,463 @@
...
+
+![](/images/bee.png)
+
+</ResourceCard>
+</Column>
+
+<Column colMd={4} colLg={4} noGutterSm>
+<ResourceCard
+subTitle="IBM rebus request tool
in Brand Center"
```
Comment (alisonjoseph) [inline src/pages/resources.mdx]: Is this the correct text or a typo?

File: src/pages/resources.mdx [hunk 5]
```diff
@@ -0,0 +1,463 @@
...
+href="https://www.ibm.com/brand/ibm-logos/business-partners#resources"
+disabled
+>
+
+![](/images/_rebus.png)
```
Comment (alisonjoseph) [inline src/pages/resources.mdx]: this image is showing up the wrong size, take a look at this page `/design/language/elements/ibm-logos/rebus/` to see how we made it the corrrect size

**Comment by greeshmareddyg** [comment]
> Great job! Just a couple things:
> 
> * The android icon looks cut off
> * Please use regular markdown list formatting rather than `ul` and `li`

fixed the above..

File: src/pages/resources.mdx (hunk 4 above)
Comment (greeshmareddyg) [reply src/pages/resources.mdx]: Corrected this.

File: src/pages/resources.mdx (hunk 5 above)
Comment (greeshmareddyg) [reply src/pages/resources.mdx]: fixed it.

**Comment by greeshmareddyg** [comment]
The ul li elements in the Logo Request section are working fine in local but on the staging the indentation looks off and the sub links are not showing bullets. we have any fix for that?

**Comment by alisonjoseph** [comment]
Just pushed a possible fix for the lists, otherwise might be a theme bug.

**Comment by greeshmareddyg** [comment]
> Just pushed a possible fix for the lists, otherwise might be a theme bug.

@alisonjoseph Its still the same. its not showing any bullets and also padding issue.

**Comment by alisonjoseph** [comment]
@greeshmareddyg the list issue is an upstream bug in the theme, should be fixed this week sometime. It shouldn't block this PR though

**Comment by greeshmareddyg** [comment]
@alisonjoseph the design team is still working on few more content changes so Please hold on to this PR for now.

**Comment by alisonjoseph** [comment]
@greeshmareddyg I added a `WIP` label, we can remove when its ready to re-review

**Comment by greeshmareddyg** [comment]
@alisonjoseph @jnm2377 @vpicone the PR is updated, could you please review.

**Comment by vpicone** [comment]
@greeshmareddyg looking good, you need alt tags in your images though

`[](myimage.png)` → `[description of my image](myimage.png)`

**Comment by greeshmareddyg** [comment]
> @greeshmareddyg looking good, you need alt tags in your images though
> 
> `[](myimage.png)` → `[description of my image](myimage.png)`

@vpicone yeah missed it. Just updated them, please check..

**Pull request merged**